user_interaction/static/themes/kinjin-theme.js

```diff
@@ -0,0 +1,7 @@
+
+function googleTranslateElementInit() {
+  new google.translate.TranslateElement({pageLanguage: 'en', layout: google.translate.TranslateElement.InlineLayout.SIMPLE}, 'google_translate_element');
+}
+
+
+<script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
```

Comment:
Ik weet niet zeker of dit hier thuis hoort.  
Git verwacht overigens ook dat bestanden eindigen met een lege regel.

Reply:
Als het goed is volledig commented. Puur als placeholder